src/iop/colortransfer.c

```diff
@@ -112,6 +112,11 @@ int flags()
   return IOP_FLAGS_DEPRECATED | IOP_FLAGS_ONE_INSTANCE | IOP_FLAGS_PREVIEW_NON_OPENCL;
 }
 
+const char *deprecated_msg()
+{
+  return _("This module is deprecated.");
```

Comment:
Use color mapping module instead

(color mapping is basically color transfer v2)

Reply:
see PR #6929 thanks for the info !